dashboard/test/models/pd/application/facilitator1920_application_test.rb

```diff
@@ -119,10 +130,19 @@ class Facilitator1920ApplicationTest < ActiveSupport::TestCase
         application = build :pd_facilitator1920_application, form_data_hash: application_hash, course: course
 
         answers = application.full_answers
-        refute answers.key? :csf_availability
-        refute answers.key? :csf_partial_attendance_reason
-        assert answers.key? :csd_csp_fit_availability
-        assert answers.key? :csd_csp_teachercon_availability
+
+        [
+          :csf_good_standing_requirement,
+          :csf_summit_requirement,
+          :csf_workshop_requirement,
+          :csf_community_requirement
+        ].each {|x| refute answers.key? x}
+        [
+          :csd_csp_lead_summer_workshop_requirement,
```

Comment:
add `csd_csp_good_standing_requirement` and double check against final version of required fields